example_data/sample_bundle.json

```diff
@@ -470,34 +533,77 @@
             ]
           }
         ],
-        "valueString": "Next generation sequence analysis indicates that General Manager is heterozygous for the MECP2 c.953G>C p.(Arg318Pro) likely pathogenic variant that has been confirmed by Sanger sequence analysis"
+        "effectiveDateTime": "2020-10-22",
+        "note": [
+          {
+            "authorString": "genomic interpretation",
+            "text": "Next generation sequence analysis indicates that Maria321 BrighGosh321 is heterozygous for the SMC1A c.1263del p.(Lys422Serfs*28) pathogenic variant that has been confirmed by Sanger sequence analysis (see technical information below)."
+          }
+        ],
+        "valueCodeableConcept": {
+          "coding": [
+            {
+              "system": "http://loinc.org",
+              "code": "LA6577-6",
+              "display": "Negative"
+            }
+          ],
+          "text": "report finding"
+        }
       },
       "request": {
-        "method": "POST",
-        "url": "Observation"
+        "method": "PUT",
+        "url": "Observation?_id=20RG-205G0047-overall-interpretation"
       }
     },
     {
-      "fullUrl": "ServiceRequest/c87a7463-19c1-6a0d-0feb-d845d0dfae43",
+      "fullUrl": "ServiceRequest/1775336d-71ea-cd05-49a3-e80e966e1277",
       "resource": {
         "resourceType": "ServiceRequest",
-        "id": "c87a7463-19c1-6a0d-0feb-d845d0dfae43",
+        "id": "1775336d-71ea-cd05-49a3-e80e966e1277",
         "meta": {
           "profile": [
             "http://hl7.org/fhir/StructureDefinition/servicerequest"
           ]
         },
+        "text": {
+          "status": "generated",
+          "div": "<div xmlns=\\\"http://www.w3.org/1999/xhtml\\\">Lab Procedure from RedCap FHIR app</div>"
+        },
         "status": "active",
         "intent": "order",
-        "supportingInfo": [
-          {
-            "reference": "PlanDefinition/38018b47-b29a-8b06-daf6-6c5f2577bffa"
-          }
+        "instantiatesCanonical": [
+          "PlanDefinition/20RG-205G0047-plan-definition"
```

Comment:
just a note to me that this is new

Reply:
we had it to be supportingInfo. Reverted